charts/hono-adapter-kura/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+appVersion: "1.0-M5"
+description: A Helm chart for the Hono-Kura adapter
+name: hono-adapter-kura
+version: 0.1.0
```

Comment:
Icon is missing, you could use: https://raw.githubusercontent.com/eclipse/hono/master/site/documentation/static/images/logo_300px276px.png

Reply:
ok